docs/Releasing.md

```diff
@@ -4,16 +4,38 @@ Open Enclave SDK Release Procedures
 This document covers how we handle creating a release. Let's look at it from an
 example:
 
+Selecting a Release Manager
+---------------------------
+
+A committer can be nominated to become the release manager by drafting a pull
+request that sets the "Release" tag to the "Area" of one of the committers in
+[Committers.md](Committers.md). The submitter of that nomination should then
+send a link to that pull request through email to all of the committers for open
+discussion. This discussion & PR should be open for at least 3 days, and people
+can make their votes known in the pull request itself by either approving the
+pull request or requesting changes. If there is any opposition from the
+committers after 3 days, then the Committee Governance Committee should take up
+this issue as high priority over email and decide whether to accept or reject
```

Comment:
Email? Or the GitHub issue? I really don't want us splintering into yet more tools.

Reply:
Email is a private forum for the committee members to discuss an urgent issue. We could also call for a short-notice CGC meeting, or put it off until the next CGC meeting.